alacritty_terminal/src/event_loop.rs

```diff
@@ -313,17 +335,12 @@ where
             let mut state = State::default();
             let mut buf = [0u8; READ_BUFFER_SIZE];
 
-            let mut tokens = (0..).map(Into::into);
-
-            let poll_opts = PollOpt::edge() | PollOpt::oneshot();
-
-            let channel_token = tokens.next().unwrap();
-            self.poll.register(&self.rx, channel_token, Ready::readable(), poll_opts).unwrap();
+            let poll_opts = polling::PollMode::EdgeOneshot;
```

Comment:
Thinking about it, do we really need `OneShot`? Shouldn't we be fine with just `Edge`?

Reply:
Though, from what I see now the main reason for `OneShot` is because we change between read/write modes.

Reply:
Looks like everything still works on Linux under `Oneshot`.

Reply:
I was curious to have just `Edge` and remove `OneShot`.

Reply:
That seems to work properly as well.

Reply:
I mean, it's level, so it should work...